### PR TITLE
improve options.lua

### PR DESCRIPTION
**Pull request by figuld**
replace vim.cmd; divide `options` into sections; minor improvements

### CODE REVIEW

**Review by bezlant — APPROVED** [review]
These are great changes! Thanks!
The `vim.opt.formatoptions:remove("cro")` doesn't seem to work either,
I found a way to fix it 
```
 vim.api.nvim_create_autocmd("BufEnter", {
	command = "set formatoptions-=cro",
})
```

File: lua/user/options.lua [hunk 1]
```diff
@@ -1,46 +1,51 @@
...
+  -- Other
+  scrolloff = 8,                           -- scroll vertically when the cursor is <8 columns from the end
+  sidescrolloff = 8,                       -- horizontally
+  whichwrap = "bs<>[]hl",                  -- which "horizontal" chars are allowed to travel to prev/next line, see :help 'whichwrap'
 }
 
-vim.opt.shortmess:append "c"
+vim.opt.shortmess:append("c") -- don't give Ctrl-P,Ctrl-N messages
+vim.opt.runtimepath:remove("/usr/share/vim/vimfiles") -- to separate vim plugins from neovim on Arch
+vim.opt.iskeyword:remove("-") -- threat a-word, word- as one w
```
Comment (pmav99) [inline lua/user/options.lua]: `s/threat/treat/`

**Comment by figuld** [comment]
> I found a way to fix it

This piece of code is already in autocommands.lua

**Comment by figuld** [comment]
> These are great changes! Thanks! The `vim.opt.formatoptions:remove("cro")` doesn't seem to work either

Figured out why and fixed it, now you don't have to use autocmd

**Review by bezlant — COMMENTED** [review]
I tried it and it didn't work for me for some reason (only removed the "o", I checked with ":set formatoptions?")

**Comment by figuld** [comment]
Seems to be no longer needed.

**Pull request closed without merging**